### PR TITLE
Fix for https://github.com/wenzhixin/bootstrap-table/issues/938

### PR DESCRIPTION
**Pull request by dimbslmh**
Partial fix for https://github.com/wenzhixin/bootstrap-table/issues/938.

1. Multiple Sort
2. Hide/Show column

Result: No assigned arrows set.

This fix retains assigned sortable arrows onColumnSwitch.

### CODE REVIEW

**Comment by djhvscf** [comment]
So can you add the onResetView method, please?

**Comment by djhvscf** [comment]
Fix #938

**Comment by dimbslmh** [comment]
Don't approve yet.

**Comment by dimbslmh** [comment]
This should be working OK now: https://jsfiddle.net/dimbslmh/s2h1gzj9/

File: src/bootstrap-table.js [hunk 1]
```diff
@@ -77,7 +77,7 @@
             $('body').append(outer);
 
             w1 = inner[0].offsetWidth;
-            outer.css('overflow', 'scroll');
+            outer.css('overflow', 'scroll');n
             w2 = inner[0].offsetWidth;
```
Comment (djhvscf) [inline src/bootstrap-table.js]: Typo here

File: src/bootstrap-table.js [hunk 2]
```diff
@@ -77,7 +77,7 @@
             $('body').append(outer);
 
             w1 = inner[0].offsetWidth;
-            outer.css('overflow', 'scroll');
+            outer.css('overflow', 'scroll');n
```
Comment (djhvscf) [inline src/bootstrap-table.js]: Typo here

**Comment by djhvscf** [comment]
Can you fix the typo?

**Comment by dimbslmh** [comment]
Fixed the typo.

**Comment by djhvscf** [comment]
Also.. can you add the documentation for onResetView method, please?

**Comment by dimbslmh** [comment]
I've added the event to the english documentation. I don't know spanish/chinese.

**Comment by wenzhixin** [comment]
Need to rebase.

**Comment by dimbslmh** [comment]
D:\GitHub\bootstrap-table [master]> git status
    On branch master
    Your branch is up-to-date with 'origin/master'.

    nothing to commit, working directory clean

"This pull request contains merge conflicts that must be resolved."

What do I do next?

**Comment by djhvscf** [comment]
You need rebase this PR

**Comment by wenzhixin** [comment]
I think there is something wrong with the rebase, can we recreate a new PR?

**Comment by wenzhixin** [comment]
Fixed the PR use command line.

**Pull request closed without merging**